site/cypress/e2e/Cypress-Gradeable/bulkupload.spec.js

```diff
@@ -24,5 +24,53 @@ describe('Test cases revolving around bulk uploading', () => {
             cy.get('#num_pages').should('be.visible');
             cy.get('#submission-mode-warning > .warning').should('have.text', 'Warning: Submitting files for bulk upload!');
         });
+
+        it(`${user} should be able to upload and split by QR code`, () => {
+            cy.login(user);
+            cy.visit(['sample', 'gradeable', 'bulk_upload_test']);
+            cy.get('#radio-bulk').click();
+            cy.get('#use-qr').check();
+            cy.get('#bulk-upload-file').selectFile('cypress/fixtures/bulk_upload_qr.pdf', { force: true });
```

Comment:
I ran the tests in CI and its failing here. These tests should pass before being merged. Check out [the docs](https://submitty.org/developer/testing/cypress) on how to run & debug locally.